.github/workflows/deploy-to-devnet.yml

```diff
@@ -60,21 +50,73 @@ jobs:
-        env:                                                                                                          
-          RELEASE_IMAGE: public.ecr.aws/p6e8q1z1/aleph-node:${{ steps.vars.outputs.sha_short }}
-          REGIONS_AWS: 'eu-central-1'
+      - name: Setup kubectl
+        uses: azure/setup-kubectl@v2.0
+
+      - name: Install yq
+        id: setup-yq
+        uses: shiipou/setup-yq-action@v2.0.1
+
+      - name: Run fork-off update
+        env:
+          RELEASE_TAG: ${{ steps.vars.outputs.sha_short }}
+
         run: |
-          IFS="," read -a region_array <<< ${{ env.REGIONS_AWS }}
-          export aleph_path=$(pwd)
-          for i in "${region_array[@]}"; do
-            
-            # Deploy new image version for validator
-            cd ${aleph_path}/aleph-apps/aleph-node-validators/overlays/devnet/${i}
-            kustomize edit set image "aleph-node-validator-image-placeholder=${{ env.RELEASE_IMAGE }}"
+          #!/bin/bash
+
+          SYSTEM_VERSION=$(curl -s -H "Content-Type: application/json" -d '{"id":1, "jsonrpc":"2.0", "method": "system_version"}' https://rpc.test.azero.dev | jq -r '.result')
+          SUFFIX="-x86_64-linux-gnu"
+          TMP_COMMIT_ID=${SYSTEM_VERSION/%$SUFFIX}
+          COMMIT_ID=${TMP_COMMIT_ID: -7}
+          echo $COMMIT_ID
+
+          if aws s3api head-object --bucket alephzero-devnet-eu-central-1-keys-bucket --key forks/chainspec-${COMMIT_ID}.json; then
+            echo "Chainspec exists"
+            IF_CHAINSPEC_EXISTS=true
+            aws s3 cp s3://alephzero-devnet-eu-central-1-keys-bucket/forks/chainspec-${COMMIT_ID}.json chainspec.skeleton.json
+          else
+            echo "Chainspec doesn't exist"
+            IF_CHAINSPEC_EXISTS=false
+
+            docker run -it --env RUST_BACKTRACE=1 --entrypoint "/usr/local/bin/aleph-node" public.ecr.aws/p6e8q1z1/aleph-node:${COMMIT_ID} bootstrap-chain --raw --base-path /data --chain-id a0dnet1 --account-ids 5D34dL5prEUaGNQtPPZ3yN5Y6BnkfXunKXXz6fo7ZJbLwRRH,5GBNeWRhZc2jXu7D55rBimKYDk8PGk8itRYFTPfC8RJLKG5o,5Dfis6XL8J2P6JHUnUtArnFWndn62SydeP8ee8sG2ky9nfm9,5F4H97f7nQovyrbiq4ZetaaviNwThSVcFobcA5aGab6167dK,5DiDShBWa1fQx6gLzpf3SFBhMinCoyvHM1BWjPNsmXS8hkrW,5EFb84yH9tpcFuiKUcsmdoF7xeeY3ajG1ZLQimxQoFt9HMKR,5DZLHESsfGrJ5YzT3HuRPXsSNb589xQ4Unubh1mYLodzKdVY,5GHJzqvG6tXnngCpG7B12qjUvbo5e4e9z8Xjidk3CQZHxTPZ,5CUnSsgAyLND3bxxnfNhgWXSe9Wn676JzLpGLgyJv858qhoX,5CVKn7HAZW1Ky4r7Vkgsr7VEW88C2sHgUNDiwHY9Ct2hjU8q --sudo-account-id 5F4SvwaUEQubiqkPF8YnRfcN77cLsT2DfG4vFeQmSXNjR7hD > chainspec.skeleton.json
 
+            aws s3 cp chainspec.skeleton.json s3://alephzero-devnet-eu-central-1-keys-bucket/forks/chainspec-${COMMIT_ID}.json
+          fi
+
+          docker run -it -v $(pwd):/app public.ecr.aws/p6e8q1z1/fork-off:latest --http-rpc-endpoint=https://rpc.test.azero.dev --initial-spec-path=chainspec.skeleton.json --combined-spec-path=chainspec.json
```

Comment:
Add:
--pallets-keep-state=Aura,Aleph,Sudo,Staking,Session,Elections

These are the defaults but it never hurts to be verbose, and chances are there will be soon more pallets joining, with their own state

Reply:
If there will be more pallets joining I'd rather do it by updating fork-off, which would automatically update this workflow(it uses latest image of fork-off tool). So leaving it as default is beneficial.